src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
```

Comment:
Also, be more specific with variable names
```suggestion
  const dailyPrice = 40;
```